src/features/modals/CreateToken/WrappedInput/WrappedInput.tsx

```diff
@@ -0,0 +1,48 @@
+//- React Imports
```

Comment:
Nice solution - has been a bit of a lag on the new Input component in zUI because we were waiting for design to be confirmed. Should be merged and we should be able to get rid of `WrappedInput` soon - I'll jot it down so we remember to do it.